python/pyspark/sql/connect/column.py

```diff
@@ -15,14 +15,15 @@
 # limitations under the License.
 #
 import uuid
-from typing import cast, get_args, TYPE_CHECKING, Callable, Any
+from typing import cast, TYPE_CHECKING, Callable, Any
 
 import json
 import decimal
 import datetime
 
 import pyspark.sql.connect.proto as proto
-from pyspark.sql.connect._typing import PrimitiveType
+
+primitive_types = (bool, float, int, str)
```

Comment:
list the types here to avoid a `circular import`, since `_typing` also import column

Reply:
You can import this in the function in that case. e.g.)

```diff
  def _bin_op(
      name: str, doc: str = "binary function", reverse: bool = False
  ) -> Callable[["Column", Any], "Expression"]:
+     from pyspark.sql.connect._typing import PrimitiveType
      def _(self: "Column", other: Any) -> "Expression":
```

Reply:
thanks, let me update